### ISSUE TITLE
About Hyper-Parameters

### ISSUE THREAD

**Issue by Oceanlib** [megvii-research/ECCV2022-RIFE#283]
Hi, 
Thanks for your code!
I see that the hyper-parameters in this released code are quite different with those reported in paper(lr, weight decay and batch size).  Which one can be used to reproduce the results?

**Comment by hzwer** [comment]
Hi, these two settings can get similar effect in Vimeo90K benchmark. The main difference is that we 10x the weight decay which may make the training more stable.

**Comment by Oceanlib** [comment]
Thank you for your reply.
